lib/std/buffer.zig

```diff
@@ -149,7 +143,27 @@ pub const Buffer = struct {
     }
 
     pub fn print(self: *Buffer, comptime fmt: []const u8, args: var) !void {
-        return std.fmt.format(self, error{OutOfMemory}, Buffer.append, fmt, args);
+        const BufferStream = struct {
```

Comment:
This is indicating we should have a standard way of converting Buffer into an OutStream.

Reply:
https://github.com/ziglang/zig/pull/4665/files#diff-d55cbcbbfea0b63c681837c0d91763d5R212-R231